content/blog/best-ui-ux-design-tools-2020/index.md

```diff
@@ -0,0 +1,530 @@
+---
+title: "Best Free UI/UX Design Tools/Resources 2020"
+date: "2020-08-13"
+coverImage: "ux-ui.jpg"
+author: "Hitesh Kumawat"
+tags: ["Design", "UX", "Tools"]
+description: "As a designer, often wondered which new tools and utilities, other designers are using ? So here is a list of all the tools and utilities famous around the world in designers."
+---
+# **Contents**
+
+1. Inspiration
+2. Colour
+3. Components
+4. Mockups
+5. Design System
+6. Font
+7. Prototyping Tools
+8. Free Stock Photos
+9. Tools that speed up your workflow
+10. Collaboration Tools
+
+## 1. **Inspiration**
+
+These are the websites where the world’s top design talents share their works and ideas.
+
+### 1. **Behance**
```

Comment:
we can make these bullet points or with heading ####, numbering is not looks good.